dashboard/app/models/ability.rb

```diff
@@ -167,6 +167,8 @@ def initialize(user)
       # all signed in users can get their level source
       can :get_level_source, UserLevel
 
+      can :evaluate, :openai_evaluate
+
```

Comment:
I updated this because since we're running the AI evaluation "behind the scene" we actually want anyone who works on the target levels to be able to hit the evaluate endpoint, not just teachers.